scss/bitstyles/ui/SectionHeading.js

```diff
@@ -0,0 +1,52 @@
+export const Label = () => {
+  const span = document.createElement('span');
```

Comment:
Maybe call this element something like `textLabel`, so it’s clear what we’re handling? Perhaps the function should be called that too